src/index.ts

```diff
@@ -155,6 +173,14 @@ export function withSpid(
       );
     })
     .map(spidStrategy => {
+      if (appConfig.startupIdpsMetadata) {
+        loadSpidStrategyOptions()
```

Comment:
This is not clear. It looks like  we're calling `loadSpidStrategyOptions()` twice in case `appConfig.startupIdpsMetadata` is defined. First time here: https://github.com/pagopa/io-spid-commons/pull/24/files#diff-f41e9d04a45c83f3b6f6e630f10117feR152
Why ?